magniv/build/build.py

```diff
@@ -66,8 +66,7 @@ def get_tasks_from_file(filepath: str, root: str, req: str, used_keys: Dict) ->
                 }
                 for kw in decorator.keywords:
                     info[kw.arg] = kw.value.value
-                missing_reqs = list({"schedule"} - set(info))
-                if len(missing_reqs) > 0:
+                if missing_reqs := list({"schedule"} - set(info)):
```

Comment:
@seanabreau  what is this notation? I have never seen it lol

Reply:
it's called a walrus operator allows you to assign a value to a variable within the same expression https://docs.python.org/3/whatsnew/3.8.html

Reply:
ah yes, i should update the badge